src/qt/sendcoinsdialog.cpp

```diff
@@ -536,7 +554,9 @@ void SendCoinsDialog::setBalance(const interfaces::WalletBalances& balances)
 {
     if(model && model->getOptionsModel())
     {
-        ui->labelBalance->setText(BitcoinUnits::formatWithUnit(model->getOptionsModel()->getDisplayUnit(), balances.balance));
+        ui->labelBalance->setText(BitcoinUnits::formatWithUnit(model->getOptionsModel()->getDisplayUnit(), balances.balance + balances.mweb_balance));
```

Comment:
This looks great!
![image](https://user-images.githubusercontent.com/24623306/102709808-b204c600-427b-11eb-88c0-b553f291b8fe.png)